src/client/utils/browser.js

```diff
@@ -22,40 +22,56 @@ function calculateBrowserAndVersion (userAgent) {
     };
 }
 
+
+export var isIE11;
```

Comment:
I doubt user agent information changes for the different windows of the **same** browser. Why can't we just get rid of `init` and `export default` object with all this properties?